docs/pages/enroll-resources/application-access/guides/vnet.mdx

```diff
@@ -140,28 +140,39 @@ $ tsh login --proxy=leaf.example.com --user=email@example.com
 
 ### Accessing web apps through VNet
 
-VNet does not officially support [web apps](connecting-apps.mdx) yet. However, technically any web
-app is [a TCP app](tcp.mdx), so they can be made available over VNet as well. You'll need to change
-`uri` of your application in the Application Service configuration file to use `tcp://` instead of
-`http://`. There's also a couple of caveats:
+VNet does not officially support [web apps](connecting-apps.mdx) yet.
+However, since all web apps are served over TCP, it's possible to convert a web
+app to [a TCP app](tcp.mdx) to make it available via VNet.
+You'll need to change the `uri` of the application in the app resource or
+Application Service configuration file to use `tcp://` instead of `https://`.
```

Comment:
```suggestion
You'll need to change the `uri` of the application to use `tcp://` instead of `https://`.
```

I'm not sure if we need to mention both ways where you can define an app, it makes the sentence harder to read. Alternatively, we could do something like this:


```suggestion
You'll need to change the `uri` of the application to use `tcp://` instead of
`https://` in the app resource or Application Service configuration file.
```